accelerator/common_core.h

```diff
@@ -64,6 +66,18 @@ int cc_logger_release();
 status_t ta_generate_address(const iota_config_t* const iconf, const iota_client_service_t* const service,
                              ta_generate_address_res_t* res);
 
+typedef struct {
+  const iota_config_t* iconf;
+  const iota_client_service_t* service;
+  ta_generate_address_res_t* res;
+} ta_generate_address_args_t;
+
+static const struct itimerval TA_TIMER_TIMEOUT = {.it_interval = {.tv_sec = 0, .tv_usec = 0},
+                                                  .it_value = {.tv_sec = 50, .tv_usec = 0}};
+
+status_t ta_generate_address_helper(const iota_config_t* const iconf, const iota_client_service_t* const service,
```

Comment:
If we are going to introduce a helper function, we shall be aware of its explicit applications.